pre_commit_hooks/destroyed_symlinks.py

```diff
@@ -34,9 +34,9 @@ def find_destroyed_symlinks(files: Sequence[str]) -> Sequence[str]:
             ) = splitted
             path = ' '.join(path_splitted)
             if (
-                mode_HEAD == PERMS_LINK
-                and mode_index != PERMS_LINK
-                and mode_index != PERMS_NONEXIST
+                mode_HEAD == PERMS_LINK and
+                mode_index != PERMS_LINK and
+                mode_index != PERMS_NONEXIST
```

Comment:
https://www.flake8rules.com/rules/W504.html
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
you don't need to lecture me, I maintain both pycodestyle and flake8 :)

Reply:
if you're curious about the history here though, pep8 has gone back and forth on this twice now and I'm not interested in the churn -- plus I think it's much more important that the conditions line up than the operators

and fortunately you don't need to have a discussion about it at all because the formatters enforce it!

Reply:
Nice :)
Anyway, for some reason, locally pre-commit didn't complain about this part.

Reply:
weird!  it might've been due to a rebase (since `.git/hooks/pre-commit` doesn't fire on a `git rebase --continue`)?

Reply:
> plus I think it's much more important that the conditions line up than the operators

That's why I used `all(...)` in the first place.

> and fortunately you don't need to have a discussion about it at all because the formatters enforce it!

No offences here, I've started discussion mainly because I was surprised with different behaviour between local and CI versions. The reason seems to be the fact that hook uses system autopep8 and, apparently, we have different versions.

Reply:
> weird! it might've been due to a rebase (since .git/hooks/pre-commit doesn't fire on a git rebase --continue)?

I've launched `tox` manually (which launches pre-commit for all project).

My versions: autopep8 1.5.4 (pycodestyle: 2.6.0)

Reply:
there's probably some global configuration you have -- it doesn't use system `autopep8` (that's most of the point of `pre-commit` in that you don't have to install things globally!)